src/api/vdom.js

```diff
@@ -295,6 +295,21 @@ function newElementVoid (tag, ...args) {
 // Text override ensures their calls can queue if using function helpers.
 const newText = wrapIdomFunc(text);
 
+export function child (ch) {
+  const ctype = typeof ch;
+  if (ctype === 'function') {
+    ch();
+  } else if (ctype === 'string' || ctype === 'number') {
+    newText(ch);
+  } else if (Array.isArray(ch)) {
+    ch.forEach(child);
+  } else if (ctype === 'object' && ch.$$typeof) {
```

Comment:
hmm so you don't like commenting code. Me neither. Next time maybe it could be better to name things appropriately via function helper or variable.

What I would like to see when reading the code would be something like this:
```js
export function child(ch){
 const ctype = typeof ch;

 if (isFunctionHelper(ctype)) {
  ch();
  } else if (isTextNode(ctype)) {
     newText(ch);
  } else if (Array.isArray(ch)) {
     ch.forEach(child);
  } else if (isReactNode(ctype)) {
     // ...
  }

}
 
 function isFunctionHelper(child){
   return ctype === 'function'
 }
 function isTextNode(child){
   return ctype === 'string' || ctype === 'number';
 }
  function isReactNode(child){
    return ctype === 'object' && ch.$$typeof
  }
}
```

WDYT?